apps/src/code-studio/pd/application_dashboard/detail_view/facilitator_scoring_fields.jsx

```diff
@@ -0,0 +1,229 @@
+        </p>
+      </div>
+    )
+  },
+  experienceTeachingThisCourse: {
+    title: 'Content Knowledge',
+    rubric: (
+      <div>
+        <p>
+          0: No experience teaching CS
+        </p>
+        <p>
+          3: Has taught CS, but not this curriculum
+        </p>
+        <p>
+          5: Has taught this curriculum to the appropriate grade level
+        </p>
+      </div>
+    )
+  },
+  completedPd: {
+    title: 'Content Knowledge',
+    rubric: (
+      <div>
+        <p>
+          0: Has not participated in the Professional Learning Program for this curriculum
+        </p>
+        <p>
+          3: Has participated in the Professional Learning Program for a different
+          curriculum
+        </p>
+        <p>
+          5: Has participated in the Professional Learning Program for this curriculum
+        </p>
+      </div>
+    )
+  },
+  whyShouldAllHaveAccess: {
+    title: 'Equity',
+    rubric: (
+      <div>
+        <p>
+          0: Does not articulate why CS should be accessible to all teachers
+        </p>
+        <p>
+          3: Articulates the importance of CS, but is not clear about accessibility for
+          all teachers regardless of background or certification.
+        </p>
+        <p>
+          5: Clearly articulates the importance of CS accessibility for all teachers
+          regardless of background or certification.
+        </p>
+      </div>
+    )
+  },
+  skillsAreasToImprove: {
+    title: 'Growth minded',
+    rubric: (
+      <div>
+        <p>
+          0: Does not describe two strengths and two areas of improvement
+        </p>
+        <p>
+          3: Describes fewer than four total strengths and areas of improvement
+        </p>
+        <p>
+          5: Describes at least two strengths and two areas of improvement
+        </p>
+      </div>
+    )
+  },
+  inquiryBasedLearning: {
+    title: 'Teaching Experience',
+    rubric: (
+      <div>
+        <p>
+          0: Does not provide a clear definition or example of inquiry-based learning
+        </p>
+        <p>
+          3: Defines inquiry-based learning but has not led an inquiry-based activity.
+        </p>
+        <p>
+          5: Defines inquiry-based learning and provides a concrete personal example of an
+          inquiry-based activity they have led
+        </p>
+      </div>
+    )
+  },
+  whyInterested: {
+    title: 'Program Commitment',
+    rubric: (
+      <div>
+        <p>
+          0: Describes neither what they want to learn nor the impact they want to make,
+          OR states that they want to learn because their intention is to become a trainer
+          primarily for their district
+        </p>
+        <p>
+          3: Describes either what they want to learn OR the impact they want to make
+        </p>
+        <p>
+          5: Describes what they want to learn AND the impact they want to make.
```

Comment:
minor comment: there is inconsistent use of periods at the end of sentences for grading. Do we want there to always or never be a period?

Reply:
I'll remove all periods